src/pudl/settings.py

```diff
@@ -647,6 +648,76 @@ class EtlSettings(BaseSettings):
     pudl_in: str = pudl.workspace.setup.get_defaults()["pudl_in"]
     pudl_out: str = pudl.workspace.setup.get_defaults()["pudl_out"]
 
+    @root_validator(pre=False)
+    def raw_table_validation_ferc1(cls, field_values):
+        """Require the presence of raw FERC1 tables needed for PUDL table creation."""
+        # only check if we are actually loading any pudl tables
```

Comment:
I'm confused by this comment. It seems like you're checking whether any FERC DBF or XBRL tables are being loaded into their respective DBs, not whether any PUDL tables are being loaded. Do we want to do this validation if no FERC 1 tables are being loaded into PUDL? In that case then any set of FERC to SQLite settings would be valid, since no tables are required downstream.

Reply:
* If any PUDL tables are being loaded, we use that list of PUDL tables to check for the required raw tables (via the table map).
* If we have any ferc1_to_sqlite settings, we check if those settings include the required tables

Right now, when we check if we have the required tables, we use a `difference` - not a `symetrical_difference`. We want to know if there are required tables that aren't in our ferc_to_sqlite settings. This means we could load a PUDL db with zero ferc-derived tables alongside ferc_to_sqlite tables.